package.json

```diff
@@ -43,7 +43,7 @@
   "author": "GitHub, Inc.",
   "license": "MIT",
   "dependencies": {
-    "@primer/octicons-react": "^13.0.0",
+    "@primer/octicons-react": "^16.1.0",
```

Comment:
The new icon I'm using here (`IssueDraftIcon`) was added here https://github.com/primer/octicons/pull/614 and shipped with version 14. I'm not sure how we update dependencies in such cases.

Reply:
This is likely causing some test failures right now 😢

Reply:
Looks like the snapshots are stale. Does running `test:update` command help the tests pass?

Reply:
@rezrah looks like they are passing now with https://github.com/primer/react/pull/1604/commits/dd6b246764bd69404d5d99207469568039f3acb2